CRM/Core/BAO/Dashboard.php

```diff
@@ -357,11 +357,30 @@ public static function saveDashletChanges($columns, $contactID = NULL) {
       }
     }
 
-    // Disable inactive widgets
+    // Find dashlets in this domain.
+    $domainDashlets = civicrm_api3('Dashboard', 'get', [
+      'return' => array('id'),
+      'domain_id' => CRM_Core_Config::domainID(),
+    ]);
+
+    // Get the array of IDs.
+    $domainDashletIDs = [];
+    if ($domainDashlets['is_error'] == 0) {
+      foreach ($domainDashlets['values'] as $domainDashlet) {
```

Comment:
Could we try this instead @christianwach 

```suggestion
      $domainDashletIDs  = CRM_Utils_Array::collect('id', $domainDashlets['values'])
```

Reply:
@seamuslee001 Thanks - done in 4020bce857b24ea7fce12a69700ee185ce1ad447